ACHNBrowserUI/ACHNBrowserUI/views/settings/SettingsView.swift

```diff
@@ -205,6 +253,35 @@ struct SettingsView: View {
                     content: { self.deleteConfirmationAlert })
         }
     }
+    
+    func getNotify(title: String, subtitle: String, hour: Int, minute: Int, isRepeated: Bool) {
```

Comment:
Don’t do this here. Add notifications to `NotificationsManager` in Backend package.